### PR TITLE
arash/m-keepsafe

### PR DESCRIPTION
**Pull request by arash-fs**

### CODE REVIEW

File: src/components/elements/hero.js [hunk 1]
```diff
@@ -12,14 +12,15 @@ const Wrapper = styled.section`
...
+    max-width: ${props => props.paragraph_max_width || ''};
 `
 
 const NoImageHero = styled(Container)`
     flex-direction: column;
     align-content: space-around;
     padding: 8rem 0 10rem 0;
 `
-const Hero = ({ header, paragraph, paragraph_width }) => (
+const Hero = ({ header, paragraph, paragraph_width, paragraph_max_width }) => (
```
Comment (mustofa-binary) [inline src/components/elements/hero.js]: is it possible to use either max_width or width instead of both

File: src/pages/keep-safe/index.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import styled from 'styled-components'
 import SEO from '../../components/containers/seo'
 import { SecurityIconGrid } from './_icon-grid'
+import Show from 'components/containers/show'
```
Comment (mustofa-binary) [inline src/pages/keep-safe/index.js]: you can run npm run test, to fix all of import orders

File: src/components/elements/hero.js (hunk 1 above)
Comment (arash-fs) [reply src/components/elements/hero.js]: yes, sometimes we need to set a limit on it.

File: src/pages/keep-safe/index.js (hunk 2 above)
Comment (arash-fs) [reply src/pages/keep-safe/index.js]: Sure

**Pull request merged**